sklearn/decomposition/tests/test_pca.py

```diff
@@ -389,6 +390,12 @@ def test_pca_validation():
                                     PCA(n_components, svd_solver=solver)
                                     .fit, data)
 
```

Comment:
When constructing test in PCA, it is common to use `for solver in solver_list:` to loop through all the possible svd_solvers. There's already a loop above your code so it might be better to move your test into the loop to make sure that the code works for all svd_solvers.

Reply:
Ye, I will do it since @jnothman also suggested it